data_generation.py

```diff
@@ -79,11 +78,16 @@ def generate_spikes(self, z):
             
             if self.bump_placement == 'random':
                 rf_location = np.random.rand(self.num_neuron * self.num_ensemble, self.dim)
-                rf_location = min_z + rf * (max_z - min_z)
+                rf_location = min_z + rf_location * (max_z - min_z)
```

Comment:
above in line 73, can you split the expression over multiple lines to make it more readable? also whitespaces around '+' and after ','